tests/unit/test.errors.js

```diff
@@ -76,6 +98,17 @@ describe('errors', () => {
         });
     });
 
+    it('throws errors that are not in an array of errno', () => {
```

Comment:
I don't think you need to include this test. It doesn't introduce any new coverage since the previous one already checks that an array of mismatching codes does not catch any error at all.